StardewArchipelago/GameModifications/Modded/JunimoShopGenerator.cs

```diff
@@ -0,0 +1,504 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Xml.Schema;
+using Microsoft.Xna.Framework;
+using StardewArchipelago.Archipelago;
+using StardewArchipelago.Constants;
+using StardewArchipelago.Extensions;
+using StardewArchipelago.Locations;
+using StardewArchipelago.Stardew;
+using StardewValley;
+using StardewValley.Locations;
+using StardewValley.Menus;
+
+namespace StardewArchipelago.GameModifications.Modded
+{
+    public class JunimoShopGenerator
+    {
+        private ArchipelagoClient _archipelago;
+        private ShopStockGenerator _shopStockGenerator;
+        private StardewItemManager _stardewItemManager;
+        private PersistentStock BluePersistentStock { get; }
+        private PersistentStock GreyPersistentStock { get; }
+        private PersistentStock YellowPersistentStock { get; }
+        private PersistentStock RedPersistentStock { get; }
+        private PersistentStock OrangePersistentStock {get; }
+        private static Dictionary<string, JunimoVendor> JunimoVendors { get; set; }
+        private Dictionary<int, int> BlueItems { get; set; }
+        private static readonly List<string> BlueColors = new()
```

Comment:
If you do not intend on adding or removing items later, these should be `string[]`, not `List<string>`